internal/bgp/bgpfrr/parse.go

```diff
@@ -65,19 +73,25 @@ func parseNeighbour(vtyshRes string) (*Neighbor, error) {
 		if n.BgpState != bgpConnected {
 			connected = false
 		}
+		pfxSnt := 0
+		for _, s := range n.AddressFamilyInfo {
+			pfxSnt += s.SentPrefixCounter
+		}
 		return &Neighbor{
-			ip:        ip,
-			connected: connected,
-			localAS:   strconv.Itoa(n.LocalAs),
-			remoteAS:  strconv.Itoa(n.RemoteAs),
+			Ip:          ip,
+			Connected:   connected,
+			LocalAS:     strconv.Itoa(n.LocalAs),
```

Comment:
This is a different naming convention that is used compared to the rest of MetalLB (myasn, asn). However, I much prefer yours. Maybe we could consolidate somehow .. not sure.